src/jit/lir.cpp

```diff
@@ -129,7 +129,14 @@ GenTree* LIR::Use::User() const
 //
 bool LIR::Use::IsValid() const
```

Comment:
I suppose I should have named these checks `IsInitialized()`, as that's really what they mean. The checks you've added are good, but should be moved to a different, heavier-weight, debug-only method.